modules/basicfuncs/tests/test_basicfuncs.c

```diff
@@ -136,6 +146,23 @@ Test(basicfuncs, test_str_funcs)
 {
   assert_template_format("$(ipv4-to-int $SOURCEIP)", "168496141");
 
+  if (_is_connected_to_the_internet())
```

Comment:
I think we might want to put this in a separate test function, but this is only a nitpick.